specification/security/resource-manager/Microsoft.Security/preview/2022-11-20-preview/apiCollections.json

```diff
@@ -0,0 +1,298 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Defender for Cloud",
+    "description": "API spec for Microsoft.Security (Microsoft Defender for Cloud) resource provider",
+    "version": "2022-11-20-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
```

Comment:
>schemes [](http://example.com/codeflow?start=3&length=7)

We have a requirement for all new namespaces to be on RPaaS. Please read the documentation here and setup an onboarding meeting with Evan Hissey

Reply:
Unfortunately, Microsoft.Security RP is not an RPaaS managed UserRP as far as I know. This decision was made many years ago by the owners of this RP i.e. the MDC team.

My team's only requirement for public preview is to add our resources and APIs to the Microsoft.Security RP. Onboarding to RPaaS is out of scope for this PR, because the work involved to do that is owned by the MDC team who has overall ownership of the RP.

Reply:
Thanks for the clarifications!